### PR TITLE
navbar initial commit

### PR DESCRIPTION
**Pull request by yehtunkhine**
- added a mobile responsive nav bar using react-responsive
- didnt delete the mongodb nav bar for testing purpose

### CODE REVIEW

**Comment by AryamanAgrawal** [comment]
- Remember to request a reviewer when submitting a pull request.
- Use `NavLink` component to work with React Router like in the initial template.
- Routing has `#` in the path right now, we do not want that
- Put in dummy pages (new components) for the different tabs
- Follow code structure and practices from the initial template

File: client/src/components/navbar.js [hunk 1]
```diff
@@ -1,13 +1,72 @@
...
+    const isDesktopOrLaptop = useMediaQuery({ minWidth: 1224 })
+    const isBigScreen = useMediaQuery({ minWidth: 1824 })
+    const isTabletOrMobile = useMediaQuery({ maxWidth: 1224 })
+    const isPortrait = useMediaQuery({ orientation: 'portrait' })
+    const isRetina = useMediaQuery({ minResolution: '2dppx' })
+
+    //Menu one the Navbar
+    const Menu = () => (
+    <>
+        <p><a href="#home">Home</a></p>
```
Comment (AryamanAgrawal) [inline client/src/components/navbar.js]: Here use NavLink and remove the `#` from the routes

**Review by AryamanAgrawal — CHANGES_REQUESTED** [review]
Yup, refer to the comments above, not a lot, let's get it done right. Awesome job though!

**Comment by alexdhima** [comment]
Hey Ye, the nav bar looks great. I have one concern though. When a use first opens up the page, shouldn't the login/sign up be in the middle of the screen where users can simply type in their email and password? I'm just not sure when this navbar with the login + signup on the top right would ever be used. I guess it could be used for users that do not want to login, but is our project going to allow that feature? just putting thoughts out there

**Pull request closed without merging**